lib/resources/r4/nutrition_order.yaml

```diff
@@ -0,0 +1,85 @@
+  - name: oralDiet.type
+    required: 'No'
+    type: List of CodeableConcept
+    description: The kind of diet or dietary restriction such as fiber restricted diet or diabetic diet.
+    binding:
+      description: The kind of diet or dietary restriction.
+      terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: http://hl7.org/fhir/r4/snomedct.html
+  - name: schedule.code
+    required: 'No'
+    type: CodeableConcept
+    description: A code for the timing schedule.
+    binding:
+      description: Code for a known/defined timing pattern.
+      terminology:
+      - display: TimingAbbreviation
+        system: https://hl7.org/fhir/r4/v3/GTSAbbreviation/cs.html
+        info_link: https://www.hl7.org/fhir/valueset-servicerequest-category.html
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: http://hl7.org/fhir/r4/snomedct.html
+      - display: Millennium Frequency
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/4003
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-4003-frequency
+  - name: oralDiet.texture.modifier
+    required: 'No'
+    type: CodeableConcept
+    description: Any texture modifications (for solid foods) that should be made.
+    binding:
+      description: Texture modifications that should be made for solid food.
+      terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: http://hl7.org/fhir/r4/snomedct.html
+      - display: Millennium Diet Consistency
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/100197
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-100197-diet-consistency
+  - name: oralDiet.fluidConsistencyType
+    required: 'No'
+    type: List of CodeableConcept
+    description: The required consistency of liquids or fluids served to the patient.
+    binding:
+      description: The required consistency of fluids served to the patient.
+      terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: http://hl7.org/fhir/r4/snomedct.html
+      - display: Millennium Diet Consistency
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/100123
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-100123-beverage-consistency
+  - name: supplement.type
+    required: 'No'
+    type: CodeableConcept
+    description: The kind of nutritional supplement product required.
+    binding:
+      description: The kind of nutritional supplement required.
+      terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: http://hl7.org/fhir/r4/snomedct.html
+      - display: Millennium Dietary Supplement
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/6107
```

Comment:
EHR source id => EHRsourceid

Reply:
I think it would look nice with the spaces. I would prefer it this way. Thoughts?

Reply:
I think it's fine. Looks like Observation does this: https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/r4/observation.yaml#L96